### DISCUSSION TITLE
Windows AMD DirectML support

### DISCUSSION THREAD

**Discussion by PixifyAI** [Woolverine94/biniou#15] [Ideas]
Could we get an implementation (install_win_AMD.cmd) of DirectML for Windows as in auto1111 https://github.com/lshqqytiger/stable-diffusion-webui-directml

**Comment by Woolverine94** [comment]
Hello @PixifyAI , 

First of all, thanks for your contribution. 

From what I understand of DirectML and what I know of biniou, it doesn't seems to be an acceleration that can be integrated at this time (see [here](https://github.com/microsoft/DirectML/issues/415#issuecomment-1590256473) : pytorch 2.1.0 is the only pytorch version compatible with all pipelines used in biniou). More of this, I have no way to test it as I don't have access to AMD GPU + Windows environment.

However, this is a very good suggestion, and I will try to keep an eye on this.

**Comment by PixifyAI** [comment]
I have 6950 xt and  Ryzen 9 5950X  with 32 gig ram an 19T of HD space for
all the models you could want to use. If you want to test anything im
down to help out, just let me know.
